src/i18n/es.js

```diff
@@ -227,6 +227,7 @@ export default {
     error_invalid_domain_to_register_account: `¡Ouch! Email inválido para crear una cuenta.`,
     error_invalid_email_address: `¡Ouch! El formato del Email es incorrecto`,
     error_invalid_login: `¡Ouch! Hay un error en tu Usuario o Contraseña. Vuelve a intentarlo.`,
+    error_invalid_name: `¡Ouch! Escribe un Nombre válido.`,
```

Comment:
This same message applies for both name and lastname fields?

Reply:
i put in comments need generic message, maybe can ask Caro for the correct message we can put

Reply:
Yes, please.

Reply:
Did you get a response from Marketing side?

Reply:
non, caro its busy not respond

Reply:
mmm, let me check.